pkg/webhook/webhook.go

```diff
@@ -69,16 +70,20 @@ func (pwh *bookkeeperWebhookHandler) mutateBookkeeperManifest(ctx context.Contex
 }
 
 func (pwh *bookkeeperWebhookHandler) mutateBookkeeperVersion(ctx context.Context, bk *bookkeeperv1alpha1.BookkeeperCluster) error {
-	configMap := &corev1.ConfigMap{}
-	err := pwh.client.Get(ctx, types.NamespacedName{Name: util.ConfigMapNameForBookieVersions(bk.Name), Namespace: bk.Namespace}, configMap)
+	// The key is the supported versions, the value is a list of versions that can be upgraded to.
+	supportedVersions := make(map[string]string)
+
+	file, err := os.Open("/tmp/config/keys")
```

Comment:
Code to read from file should be added to a separate method which can be invoked here. Lets keep the code as modular as possible.

Reply:
done